File: app/src/main/java/com/channa/mobiledatausageapp/data/response/ResultResponse.java
package com.channa.mobiledatausageapp.data.response;

import java.util.List;

public class ResultResponse {
    private String resource_id;
    private List<QuarterResponse> records;

    public ResultResponse() {
    }

    public ResultResponse(String resource_id, List<QuarterResponse> records) {
        this.resource_id = resource_id;
        this.records = records;
    }

    public String getResource_id() {
        return resource_id;
    }

    public void setResource_id(String resource_id) {
        this.resource_id = resource_id;
    }

    public List<QuarterResponse> getRecords() {
        return records;
    }

    public void setRecords(List<QuarterResponse> records) {
        this.records = records;
    }
}

File: app/src/main/java/com/zak/datauses/data/response/ResultResponse.kt
package com.zak.datauses.data.response

class ResultResponse {
    var resource_id: String? = null
    var records: List<QuarterResponse>? = null

    constructor() {}
    constructor(resource_id: String?, records: List<QuarterResponse>?) {
        this.resource_id = resource_id
        this.records = records
    }
}